future-apps/payroll/contracts/Payroll.sol

```diff
@@ -331,14 +329,26 @@ contract Payroll is EtherTokenConstant, IsContract, AragonApp { //, IForwarder {
         emit DetermineAllocation(employeeIds[msg.sender], msg.sender);
     }
 
+    /**
+     * @dev Withdraw payment by employee (the caller). The amount owed since last call will be transferred.
```

Comment:
It is the specified amount the one that will be transferred instead of the owed one, right?

Reply:
Yes, that's the idea.